### ISSUE TITLE
Test accessibility features and research mobile app accessibility concerns

### ISSUE THREAD

**Issue by li-poltorak** [cassieesposito/fertilityTracking#7]
Consider setting up accessibility testing when we set up basic unit and UI tests, we should be considering accessibility needs from the start.
